CONTRIBUTING.md

```diff
@@ -27,6 +41,10 @@ case you should explain why in the merge commit.
 [Junio](https://gitster.livejournal.com/42247.html) and
 [Linus](http://yarchive.net/comp/linux/git_merges_from_upstream.html).
 
+You can use `make style` to help conform to coding conventions of the
```

Comment:
Do we want to mention that it is sometimes nice to avoid making style changes in a portion of the code that is being updated in a different PR?